### PR TITLE
Feature/pelagos 3793 add pods

### PR DESCRIPTION
**Pull request by praneethpr**
Please merge PELAGOS-3792 before merging this

### CODE REVIEW

**Comment by mickel1138** [comment]
Missing (or wrong path) applicaiton_edit.png, delete.png, Person.js

**Comment by mickel1138** [comment]
This no longer exists:
https://github.com/griidc/pelagos/blob/27a48790538479c5df44bdda0be493f02dd9748e/src/Form/ResearchGroupType.php#L109

**Comment by mickel1138** [comment]
Most all JS files missing dealing with PODS.

File: src/Controller/UI/NationalDataCenterController.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        } else {
+            $nationalDataCenter = new NationalDataCenter();
+        }
+
+        $form = $this->get('form.factory')->createNamed(null, NationalDataCenterType::class, $nationalDataCenter);
+
+        return $this->render('template/NationalDataCenter.html.twig', array(
+            'NationalDataCenter' => $nationalDataCenter,
+            'form' => $form->createView(),
+            'entityService' => $this->entityHandler
```
Comment (mickel1138) [inline src/Controller/UI/NationalDataCenterController.php]: This entityhandler is not freely available anymore. Also whoemever passed the entityhandler inside the template should be ashamed of themselves.

Comment (praneethpr) [reply src/Controller/UI/NationalDataCenterController.php]: I don't know why it is passed from all the Controllers, it is being used only in researchGroup.html.twig. I'll remove them and refactor it

**Comment by mickel1138** [comment]
linkList.js missing

**Comment by mickel1138** [comment]
Also missing publink.css and publink.js

**Comment by praneethpr** [comment]
Fixed all issues

File: src/Controller/UI/NationalDataCenterController.php [hunk 2]
```diff
@@ -49,8 +49,7 @@ public function defaultAction($id = null)
 
         return $this->render('template/NationalDataCenter.html.twig', array(
             'NationalDataCenter' => $nationalDataCenter,
-            'form' => $form->createView(),
-            'entityService' => $this->entityHandler
```
Comment (mickel1138) [inline src/Controller/UI/NationalDataCenterController.php]: Line 40 above also asks for entityHandler

Comment (praneethpr) [reply src/Controller/UI/NationalDataCenterController.php]: Fixed it

**Comment by mickel1138** [comment]
Change reference:
`An exception has been thrown during the rendering of a template ("Warning: constant(): Couldn't find constant Pelagos\Bundle\AppBundle\DataFixtures\ORM\ResearchGroupRoles::LEADERSHIP").`
in templates/template/Person.html.twig (line 68)

**Pull request merged**